static/app/stores/organizationStore.tsx

```diff
@@ -52,6 +54,10 @@ const storeConfig: OrganizationStoreDefinition = {
 
     ReleaseStore.updateOrganization(this.organization);
     LatestContextStore.onUpdateOrganization(this.organization);
+    HookStore.getCallback(
+      ROUTE_ACTIVATED_HOOK_NAME,
+      'setOrganization'
+    )(this.organization);
```

Comment:
Didn't realize HookStore only passed pure functions, not a huge fan of this since it's a little confusing but the general premise is that we should save callbacks that modify existing state. `setOrganization` / the org state on the getsentry side only needs this because it's in the same hook as the rest of the code. In theory you could have a single store on the `getsentry` side and store everything there as it updates, but I didn't want to add another store right now before we figure out what to do with global state.